.github/workflows/deploy_production.yaml

```diff
@@ -1,4 +1,4 @@
-name: "[production] Deploy"
+name: "⚠️💥💀⚡💢❗ [production] Deploy"
```

Comment:
Why emojis looks dangerous? R u scary to deploy? You main branch is not prod ready?)

Reply:
If you deploy review branches to prod a few times, you might find yourself fired, but that depends